### ISSUE TITLE
how can I get training data

### ISSUE THREAD

**Issue by yasohasakii** [gitmenonsandu/Image-Denoising-with-Deep-CNN#1]
Hi！ I've already got your src but I didnit figure out that where is the traning data and label.Does it means that I need to get data by myself?Thanks!

**Comment by harryvineeth** [comment]
In the paper, they have used STL 10-dataset     http://cs.stanford.edu/~acoates/stl10/
For our implementation, we have used CIFAR-10 dataset: https://www.cs.toronto.edu/~kriz/cifar.html
You can choose whichever suits you best.

**Comment by yasohasakii** [comment]
Great! Thanks a lot!
